client/app/domain/zone/default-ttl/domain-zone-default-ttl.html

```diff
@@ -1,40 +1,33 @@
 <div data-ng-controller="DomainZoneDefaultTTLCtrl as ctrl">
-    <div data-wizard
-         data-wizard-on-cancel="resetAction"
-         data-wizard-on-finish="updateDefaultTTL"
-         data-wizard-title="i18n.domain_tab_ZONE_default_ttl_button">
 
-        <div data-wizard-step
-             data-wizard-step-valid="!ctrl.loading && ctrl.updateDefaultTtlForm.$valid">
-
-            <div class="text-center" data-ng-if="ctrl.loading">
-                <oui-spinner></oui-spinner>
-            </div>
-
-            <div data-ng-if="!ctrl.loading">
-                <p data-ng-bind-html="tr('domain_tab_ZONE_default_ttl_heading', '<strong>' + ctrl.domain.displayName + '</strong>')"></p>
+    <oui-modal data-title="{{tr('domain_tab_ZONE_default_ttl_button')}}"
```

Comment:
Minor : could use `{{:: ... }}`

Reply:
that works with function ?

Reply:
yup we use it all around the web manager :smiley: